### ISSUE TITLE
kademo.nl is gone, all examples/reference to it or it's layers fail

### ISSUE THREAD

**Issue by rduivenvoorde** [heron-mc/heron-mc#472]
Apparently kademo.nl is not used anymore by Het Kadaster...

**Comment by rduivenvoorde** [comment]
see also: https://github.com/heron-mc/heron-mc/issues/463

**Comment by justb4** [comment]
Yes, aware of that: I have maintained kademo.nl for 8 years, but Kadaster has finished that contract and canceled the domain. As the server is the same as data.nlextract.nl I have migrated the GeoServer WMS/WFS and MapFish Print services to ws.nlextract.nl, but possibly there are still examples not  migrated, and there are older Heron versions. e.g. http://lib.heron-mc.org/heron/latest/examples/defaultnl/ has the hockeyclubs etc.
